### PR TITLE
EWPP-750: Description List paragraph.

### PR DESCRIPTION
**Pull request by 22Alexandra**

### CODE REVIEW

File: modules/oe_paragraphs_description_list/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+OpenEuropa Paragraphs Description List
+============================
+
+This module provides Description List paragraph that depends on the [OpenEuropa Content Description List Field](https://github.com/openeuropa/oe_content/modules/oe_content_description_list_field)
```
Comment (sergepavle) [inline modules/oe_paragraphs_description_list/README.md:4]: The link is not updated.

**Review by sergepavle — COMMENTED** [review]
I've left tiny comment

**Review by sergepavle — DISMISSED** [review]

File: modules/oe_paragraphs_description_list/config/install/field.field.paragraph.oe_description_list.field_oe_description_list_items.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  module:
+    - description_list_field
+id: paragraph.oe_description_list.field_oe_description_list_items
+field_name: field_oe_description_list_items
+entity_type: paragraph
+bundle: oe_description_list
+label: Items
+description: ''
+required: false
+translatable: false
```
Comment (imanoleguskiza) [inline modules/oe_paragraphs_description_list/config/install/field.field.paragraph.oe_description_list.field_oe_description_list_items.yml:16]: This field needs to be translatable.

File: tests/features/description-list.feature [hunk 3]
```diff
@@ -0,0 +1,34 @@
+@api
+Feature: Description List paragraph.
+  As a content editor
+  I need to be able to use Description List paragraphs
+  so I can display an Overview.
```
Comment (imanoleguskiza) [inline tests/features/description-list.feature:5]: "so I can display a list of term/description pairs." or something like that.

**Review by imanoleguskiza — CHANGES_REQUESTED** [review]

**Review by imanoleguskiza — APPROVED** [review]

**Review by imanoleguskiza — DISMISSED** [review]

**Review by sergepavle — APPROVED** [review]

**Review by upchuk — CHANGES_REQUESTED** [review]

**Review by sergepavle — DISMISSED** [review]

File: modules/oe_paragraphs_description_list/src/Plugin/Validation/Constraint/RequiredFields.php [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * Constraint plugin for the Description list field widget.
+ *
+ * Term and description fields should be required if any of the widget's fields
+ * are filled in.
+ *
+ * @Constraint(
+ *   id = "oe_paragraphs_description_list_required_fields",
```
Comment (upchuk) [inline modules/oe_paragraphs_description_list/src/Plugin/Validation/Constraint/RequiredFields.php:16]: Constraint IDs are typically CamelCase with first word caps as well. Check other existing ones please.

File: modules/oe_paragraphs_description_list/src/Plugin/Validation/Constraint/RequiredFields.php [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+ *   label = @Translation("Term and description fields of Description list widget become required if any of the fields are filled in", context = "Validation"),
+ *   type = "string"
+ * )
+ */
+class RequiredFields extends Constraint {
```
Comment (upchuk) [inline modules/oe_paragraphs_description_list/src/Plugin/Validation/Constraint/RequiredFields.php:21]: DescriptionListRequiredFields

**Review by upchuk — CHANGES_REQUESTED** [review]

**Review by sergepavle — APPROVED** [review]

**Review by upchuk — APPROVED** [review]

**Pull request merged**